Miscellanious changes, IPS, audio reverted, notes moved to wiki, etc.

makruger opened the pull request.

pyhalov commented:
@alarcher , please, look at it. It seems confusing

makruger commented:
Have a look here as well.....this might help a little bit.
https://github.com/makruger/oi-docs/blob/master/docs/handbook/common-tasks.md

pyhalov commented on docs/handbook/common-tasks.md:
```diff
@@ -65,15 +65,47 @@ ITEMS TO WRITE ABOUT:
+| Gstreamer | Mixer/Proxy
+| PulseAudio | Mixer/Proxy
+| OSS/OSSv4 | Open Sound System
+| Sun Audio | Solaris Audio API (Dev Audio)
```
This paragraph seems cumbersome. I don't understand all these bits deeply, but... AFAIK, Illumos Audio system provides native Sun Audio APIs. Later OSS and OSSv4 APIs were added. These are low level OS interfaces to manage audio devices. PulseAudio provides more high-level API to manage devices and mixers, provided by userland.   Gstreamer is a set of API to manage not only device access, but to perform management of streams (video/audio/etc) and encoding, so I wouldn't mix them. It would be perfect if you could ask Garrett to comment on this section, when it'll be in better state.

makruger replied:
OK, sounds reasonable. I'll run it by Garret and also refine it based on what you have just written.

I'm also not sure what level of detail we should provide. My thoughts are providing a brief introduction as well as listing and or linking to man pages. Some information about how to adjust and select things, along with some guidance for testing should be sufficient I would think.

makruger commented:
@pyhalov , @alarcher 

I've reverted the changes for audio and moved them to a Gist for further development. This will allow my other changes to move forward and keep the momentum going. 

The GIST is here: https://gist.github.com/makruger/db73ac403b1a1ea147f72f19ea836cdd

I'll run all this by Garret who looks to be the expert on the audio subsystem (Boomer, etc.).

pyhalov commented on docs/handbook/getting-started.md:
````diff
@@ -1230,24 +1230,8 @@ Note the following points about IPS packages:
-pkgrecv -s http://example.com:10000 -d /my/emacs.p5p -a -r editor/gnu-emacs
-```
+* IPS packages can only be installed from an IPS repository.
+    * IPS package repositories can be local to the system or on a remote networked system.
````
why did you remove mentions of p5p files?

makruger replied:
Hi Alp,

It was only moved towards the end of the IPS section.

Michael

On Aug 23, 2016 10:05 AM, "Alexander Pyhalov" <notifications@github.com>
wrote:

In docs/handbook/getting-started.md
<https://github.com/OpenIndiana/oi-docs/pull/42#discussion_r75872194>:

> -When the IPS system was originally conceived there was no standard on-disk format for an IPS package.
> -Hence, unlike a .rpm file, an SVR4 package, or a .nbm file, it was not possible to transfer an IPS package from system to system.
> -The remote IPS repository where the IPS package resided was the only source for the package.
> -This is because in its native state, the IPS package is not something you can normally download as a single archive file.
> -
> -Recognizing this limitation of IPS, the `.p5p` IPS archive format was developed.
> -For IPS archives, files are stored in the pax archive format, along with additional metadata, such as IPS manifest files, and a specific layout of the contents.
> -The `pkgrecv` command is used to create IPS archives.
> -
> -For example, to create a `.p5p` IPS package archive containing the package `editor/gnu-emacs` and all of its dependencies from the repository located at `http://example.com:10000`, use the following command:
> -
> -```bash
> -pkgrecv -s http://example.com:10000 -d /my/emacs.p5p -a -r editor/gnu-emacs
> -```
> +* IPS packages can only be installed from an IPS repository.
> +    * IPS package repositories can be local to the system or on a remote networked system.

why did you remove mentions of p5p files?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/OpenIndiana/oi-docs/pull/42/files/16967556883a9783be366a86f0444a9a14b2d29a#r75872194>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/AP9jzHMoeh16Ty145QDsSZUiSpSdwsrQks5qiv4IgaJpZM4Jm3Oi>
.

pyhalov replied:
Sorry, was inattentive
